### ISSUE TITLE
Bug in Conditional Config Example

### ISSUE THREAD

**Issue by jcop007** [phptek/silverstripe-sentry#53]
There should be 3 dashes above the "Only:environment: test" in the Conditional Config example on readme otherwise it doesn't work. 
 
```
---
Name: my-project-config-sentry
After:
  - 'sentry-config'
---

--- 
Only:
  environment: test
---
PhpTek\Sentry\Adaptor\SentryAdaptor:
  opts:
    # Example DSN only. Obviously you'll need to setup your own Sentry "Project"
    dsn: http://deacdf9dfedb24ccdce1b90017b39dca:deacdf9dfedb24ccdce1b90017b39dca@sentry.mydomain.nz/44
---
Except:
  environment: test
---
PhpTek\Sentry\Adaptor\SentryAdaptor:
  opts:
    # Example DSN only. Obviously you'll need to setup your own Sentry "Project"
    dsn: http://deacdf9dfedb24ccdce1b90017b39dca:deacdf9dfedb24ccdce1b90017b39dca@sentry.mydomain.nz/44
---
Only:
  environment: dev
---
PhpTek\Sentry\Adaptor\SentryAdaptor:
  opts:
    dsn: null
---
```

**Comment by jcop007** [comment]
Fixed it in PR #54

**Comment by phptek** [comment]
Merged PR#54 protip @jcop007 if your commit message includes "Fixes #1234" then GitHib will also automatically close issue #1234 once the relevant PR is merged.
